apps/frontend/src/lib/langgraph/client.ts

```diff
@@ -0,0 +1,26 @@
+import { Client } from '@langchain/langgraph-sdk';
+import { PUBLIC_LANGGRAPH_API_URL } from '$env/static/public';
+
+export function createLangGraphClient(accessToken: string): Client {
+	return new Client({
+		defaultHeaders: {
+			Authorization: `Bearer ${accessToken}`
+		},
+		apiUrl: PUBLIC_LANGGRAPH_API_URL
+	});
+}
+
+export async function createThread(
+	accessToken: string,
+	graphId: string = 'chat'
+): Promise<{ threadId: string; assistantId: string }> {
+	const client = createLangGraphClient(accessToken);
+
+	const thread = await client.threads.create();
+	const assistant = await client.assistants.create({ graphId });
+
+	return {
+		threadId: thread.thread_id,
+		assistantId: assistant.assistant_id
+	};
+}
```

Comment:
Just a comment, no changes necessary here (as it's out of scope), but a typical dev experience could be:
```ts
langgraph: LangGraphClient = LangGraphClient(accessToken)
thread: LangGraphThread = langgraph.createThread(graphId: string)
```

Note that the former is our own client, so we have control over the API layout and the latter returns our own thread type (again, allowing for abstraction). 

Whether we call these things a LangGraphClient or LangGraphThread or just Client (but namespaced, e.g. `import as`) and Thread is open to discussion.